src/main/java/org/opentripplanner/standalone/config/NodeAdapter.java

```diff
@@ -99,6 +99,10 @@ private NodeAdapter(@NotNull JsonNode node, String source, String contextPath) {
         return result;
     }
 
+    public boolean isNonEmptyArray() {
```

Comment:
NodeAdapter is pretty easy to test. Can you add one?

Reply:
Added in https://github.com/opentripplanner/OpenTripPlanner/pull/3815/commits/0695e4ad5c2bae546109bd6621feead9ea28e363